include/xtensor/xview.hpp

```diff
@@ -657,6 +657,18 @@ namespace xt
         template <class T = xexpression_type>
         enable_simd_interface<T, const_reference> flat(size_type i) const;
 
+        template <class T = xexpression_type>
+        enable_simd_interface<T, reference> front();
+
+        template <class T = xexpression_type>
+        enable_simd_interface<T, const_reference> front() const;
+
+        template <class T = xexpression_type>
+        enable_simd_interface<T, reference> back();
+
+        template <class T = xexpression_type>
+        enable_simd_interface<T, const_reference> back() const;
+
```

Comment:
I think you can use the implmeentation in `xaccessible`. `front` and `back` should always be available for views.

Reply:
Thanks, I assumed I couldn't. Done!